packages/client/components/PokerEstimatePhase.tsx

```diff
@@ -57,19 +59,22 @@ const ShowDiscussionButton = styled(PlainButton)({
   padding: 8
 })
 
+const StyledMeetingHeaderAndPhase = styled(MeetingHeaderAndPhase)<{isDesktop: boolean}>(({isDesktop}) => ({
+  width: isDesktop ? `calc(100% - ${DiscussionThreadEnum.WIDTH}px)` : '100%'
```

Comment:
@nickoferrall hey there, I moved the conditional width here to account for the drawer. If we set it on this container, then the children can just fill the container and don’t need to worry about the drawer layout.

Reply:
Nice, that makes sense